icu4c/source/i18n/unicode/simplenumberformatter.h

```diff
@@ -94,7 +91,7 @@ class U_I18N_API SimpleNumber : public UMemory {
      *
      * This setting is applied upon formatting the number. 
      *
-     * @draft ICU 73
+     * @stable ICU 73
      */
     void setMinimumIntegerDigits(uint32_t minimumIntegerDigits, UErrorCode& status);
```

Comment:
We have

```cpp
    /**
     * Truncates the most significant digits to the given maximum number of integer digits.
     *
     * This function immediately mutates the inner value.
     *
     * @draft ICU 73
     * @stable ICU 73
     */
    void truncateStart(uint32_t maximumIntegerDigits, UErrorCode& status);

    /**
     * Pads the beginning of the number with zeros up to the given minimum number of integer digits.
     *
     * This setting is applied upon formatting the number. 
     *
     * @draft ICU 73
     * @stable ICU 73
     */
    void setMinimumIntegerDigits(uint32_t minimumIntegerDigits, UErrorCode& status);
```

Why did we name the functions like this? I feel like they should either be

1. `setMaximumIntegerDigits` + `setMinimumIntegerDigits`
2. `truncateStart` + `padStart`

Reply:
@sffc can you please
1. propose what you think should be changed -- probably to align better with ICU4X / "newer convention"
2. comment here which functions should be held back at `@draft ICU 73` until you have a chance to change them (assuming the proposal goes through)?